### DISCUSSION TITLE
Issue in rebuilding FINN on U250 card for resnet50

### DISCUSSION THREAD

**Discussion by Sarika124** [Xilinx/finn#591] [General]
Hi.. I am trying to rebuild FINN using build_custom command for resenet50 targeting Alveo U250 board. I given following commands,
export FINN_XILINX_PATH=/tools/Xilinx/
export FINN_XILINX_VERSION=2020.1
export PLATFORM_REPO_PATHS=/opt/xilinx/dsa/
export XRT_DEB_VERSION=xrt_202010.2.6.655_18.04-amd64-xrt
export VITIS_PATH=tools/Xilinx/Vitis/2020.1
FINN_EXAMPLES=/home/test/finn-examples
./run-docker.sh build_custom /home/test/finn-examples/build/resnet50/

I am getting error in the 9th step of rebuilding. 
Adding screenshot of the error i am getting. please help me to solve.
![Screenshot from 2022-04-22 12-34-27](https://user-images.githubusercontent.com/104189887/164622704-f5842641-497e-4726-b750-617b1f19d306.png)

Thanks in advance!!

**Comment by gkorol** [comment]
hello,
probably the same issue [here](https://github.com/Xilinx/finn/discussions/497#discussioncomment-2472280).

**Reply by Sarika124** [reply]
Hi,
Thank you... Error BD 5-390 solved by this method..

Now I am getting error in bit stream generation... Anyone please help me to solve this..

![Screenshot from 2022-04-24 21-17-55](https://user-images.githubusercontent.com/104189887/164984724-a139b6fc-1734-4d15-99d0-5b8910a442d9.png)
